package-lock.json

```diff
@@ -1,8 +1,5130 @@
 {
```

Comment:
Looks like you accidentally changed permissions to this file to 755 instead of 644.